app/services/transfer_destroy_service.rb

```diff
@@ -4,6 +4,8 @@ def initialize(transfer_id:)
   end
 
   def call
+    raise "Transfer cannot be deleted because an audit was performed after it" unless transfer.deletable?
```

Comment:
Hmm... I'm torn here because the name of the method is generic ("can this be deleted") but the error assumes there is only one possible reason the audit couldn't be deleted. I think it's OK to just have this service call `Audit.since?` directly rather than `deletable?`

Reply:
You mean that the transfer couldn't be deleted, yes?

Reply:
@dorner Originally, I was thinking that a delete button would only appear if the transfer met the criteria for deletability and that the view might look cleaner with a `deletable?` method, but since that's not the case there doesn't seem to be a reason for it.